java/core/src/java/org/apache/orc/impl/DynamicByteArray.java

```diff
@@ -190,9 +166,7 @@ public int size() {
    */
   public void clear() {
     length = 0;
-    for(int i=0; i < data.length; ++i) {
-      data[i] = null;
-    }
+    Arrays.fill(data, null);
```

Comment:
This is orthogonal to redefine something, @deshanxiao .

Reply:
Yes, it is equivalent to the previous one.

Reply:
I will remove it.